pages/glossary/essential-text-change.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Essential Text Change
+key: essential-text-change
+unambiguous: true
+objective: false
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An _essential text change_ occurs when the modification of the text content of an HTML element is necessary to convey accurate and timely information to the user. An _essential text change_ may appear in any of the following forms:
+
+1. Providing context - Text is changed to provide users with additional information required to understand the current state of the page. For example, while searching for results, text may change to convey that the page is *loading* to provide context to the user.
+2. Live updates of important information - The content automatically updates to show the most recent information available from a data source and should not be ignored by the user. For example, updates about the users browsing sessions status or important safety information like evacuation instructions in the case of a natural disaster would constitute an Essential Text Change. Oppositely, information such as stock prices or advertisements are not important information and would not be considered an Essential Text Change. 
```

Comment:
This still feels fairly open-ended. Who decides that something is important, and that the user shouldn't ignore it? I think I'd like to see a list here of scenarios what we think are essential.

Reply:
I would agree, though the fact that it isn't objective helps with that some. I could add a few more examples so our total list would be the examples below. Would that be sufficient?

Important information:
1. Browsing session status (from definition)
2. Safety information (from definition)
3. Changing text which represents a core functionality of the web page, such as the time readings at https://www.time.gov/ or a countdown timer for an auction site (I'm not sure if this one works, as pausing it you should still be able to get something from the site)
4. A change in text when using collaborative systems, such as a chat messenger, forum, or other collaboration software.
5. A text change triggered by user input, such as selecting an option in a text-based adventure game.

Non-Important Information:
1. Stock ticker information (from definition)
2. Advertisements (from definition)
3. Live sports scores
4. "Rolling" updates such as job postings, or news article titles and links
5. Countdown timer to an event that does lose opportunity for the user, such as a countdown to the start of a sports game.

Reply:
I like that, yeah that sounds good to me.